utils/fixedlist.py

```diff
@@ -1,10 +1,23 @@
-class FixedList(list):
-    def __init__(self, maximum_item_count: int):
+from utils.db import get_redis
+
+
+class FixedList():
+    def __init__(self, name='none', maximum_item_count: int=10):
         self.max_items = maximum_item_count
+        self.name = name
 
     def append(self, *items):
-        for item in items:
-            super().append(item)
+        get_redis().rpush(self.name + ':list', *items)

```

Comment:
really big nit: what if we pre-make keys? so we don't need to generate them every time.

Reply:
I don't really see the point of this. We're still generating the keys one way or another. Having the server take care of it is fine by me.

Reply:
I mean... instead of generating a string on every execution why not make the `self._cache_key = name + ':list'` in the `__init__`.

Reply:
Ah I see, implementing that